src/utils/taskRunner.ts

```diff
@@ -183,6 +183,7 @@ export async function runTask(
   if (gulp.hasTask(name)) {
     try {
       await gulp.run(name);
+      logLn("Build is Finished", "info");
```

Comment:
Let's change that to informational message that it will contains the task name, e.g. `Task ${name} is completed`. It will be probably a good idea to add `Task ${name} is running`.

Reply:
${name} is default Show the output is become is   [info]: Task default is completed  If i change or stick with previously because  [info]: Build is completed is  a good information for the user because default task is doing the build process.

Reply:
problem then: it will show on any task as build is completed, which is misleading more. Based on build.ts files we have "build" tasks.

Reply:
No,It only show  when "build"  is finished .  **"default "  is not any task it is used as representation** whenever the default task run all the mention task will run but in **build.ts file there is only one mention task in default i.e "build"** whenever the " default " task is run it automatically run the "build"  task.And output the out/main.wasm file.

Reply:
We can also do like in  place of default task we should pass  build task because both are the same. what you say.

Reply:
> No,It only show when "build" is finished . **"default " is not any task it is used as representation** whenever the default task run all the mention task will run but in **build.ts file there is only one mention task in default i.e "build"** whenever the " default " task is run it automatically run the "build" task.And output the out/main.wasm file. There is not any misleading .We just run the build task as different manner.In place of default I also mention the "build " task and it worked.